### PR TITLE
Fix all but 4 undefined names

### LINKED ISSUE #3041: Undefined name: Which import_author() should we use in openlibrary/catalog/load.py?
### Question

 Which of the [three versions of import_author()](https://github.com/internetarchive/openlibrary/search?q=import_author) should we import into [openlibrary/catalog/load.py](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/load.py#L46)?

[ ] from openlibrary.catalog.importer.load import [import_author](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/importer/load.py#L123-L148)
[ ] from openlibrary.catalog.add_book.load_book import [import_author](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/add_book/load_book.py#L147-L175)
[ ] from openlibrary.catalog.onix.onix-import import [import_author](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/catalog/onix/onix-import.py#L83-L97)

### PR DESCRIPTION
**Pull request by cclauss**
Resolves most of #1684

Fixes #3041
https://travis-ci.org/internetarchive/openlibrary/jobs/650558602#L223

### CODE REVIEW

File: openlibrary/catalog/marc/marc_binary.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from unicodedata import normalize
 
 from openlibrary.catalog.marc import mnemonics
-from openlibrary.catalog.marc.marc_base import MarcBase, MarcException, BadMARC
+from openlibrary.catalog.marc.marc_base import MarcBase, MarcException, BadMARC, re_isbn
```
Comment (hornc) [inline openlibrary/catalog/marc/marc_binary.py]: There are too many changes in unrelated areas in this PR, this is `re_isbn` is an open issue and I believe the fix is to use `self.re_isbn`, but I haven't had time to test it properly. Splitting the out roughly by folder / code area would help reviewing and separating easy changes from ones that have more risk. Some parts of /catalog are unused, but other parts are critical to import. /scripts tend to have completely different concerns.

File: scripts/lc_marc_update.py [hunk 2]
```diff
@@ -22,6 +20,9 @@
 c = config.runtime_config['lc_marc_update']
 base_url = 'http://openlibrary.org'
 import_api_url = base_url + '/api/import'
+internal_error = '<Code>InternalError</Code>'
+no_bucket_error = '<Code>NoSuchBucket</Code>'
```
Comment (hornc) [inline scripts/lc_marc_update.py]: Where did this content come from? How do we know it's correct?

Comment (cclauss) [reply scripts/lc_marc_update.py]: The technique it to grep the repo for the word in question and see where and how it is used.  These literals were used in two or three other files.

File: openlibrary/catalog/marc/marc_binary.py (hunk 1 above)
Comment (cclauss) [reply openlibrary/catalog/marc/marc_binary.py]: Neither this class nor its superclass define a `self.re_isbn`.

**Comment by cclauss** [comment]
Let's convert this to a WIP PR like the various integration branches that we have open.  It can be a cherry picking ground for other smaller PRs that are easier to review.  I already
have too many PRs open so we will hold off on these fixes until things settle down a bit.

**Comment by cclauss** [comment]
Closing.  These items have all been fixed or will be fixed in #3054 or other single-file PRs or issues.

**Pull request closed without merging**